highfidelity/hifi
# Cases 10632 and 10633 Particle Explorer UI

[Case 10632](https://highfidelity.fogbugz.com/f/cases/10632/Particle-Emitter-Show-Properties-button-should-change-to-Hide-Properties-when-properties-are-already-visible)
Particle Emitter 'Show Properties' button should change to 'Hide Properties' when properties are already visible

[Case 10633](https://highfidelity.fogbugz.com/f/cases/10633/Particle-Effect-drop-downs-should-be-more-obvious)
Particle Effect dropdowns should be more obvious

![particleexplorer](https://user-images.githubusercontent.com/403611/34128864-08336dd6-e43a-11e7-85ef-7f0cb2a321eb.png)


## Test

1. Go to a domain where you have edit/create rights.
2. Open "Create" and create a Particle.
3. On the "Particle Explorer" tab click on the "Show Properties" button.
Expected result: 
A properties text area appears and the text on the "Show Properties" button now reads "Hide Properties".
4. Press the "Hide Properties" button.
Expected result: 
The particle properties disappear and the text of the button now reads "Show Properties".

5. Notice the tabs "General", "Emit", "Radius", etc. Try to open and close the tabs.
